docs/reference/nuctl/nuctl.md

```diff
@@ -0,0 +1,83 @@
+# Nuctl
+
+## About
+
+Nuctl is Nuclio command-line interface that provide all nuclio-features from your terminal
+
+## Download
+
+To install nuctl all you need is to simply visit Nuclio [releases page](https://github.com/nuclio/nuclio/releases)
+And download the matched binary to your platform (e.g.: `darwin` if you have `macOS`)
+
+If you are adventurous enough, you can use the following snippet
+```bash
+curl -s https://api.github.com/repos/nuclio/nuclio/releases/latest \
+			| grep -i "browser_download_url.*nuctl.*$(uname)" \
+			| cut -d : -f 2,3 \
+			| tr -d \" \
+			| wget -O nuctl -qi - && chmod +x nuctl
+```
+
+## Use
+
+Once downloaded, an informative help section is available for each command using `nuctl --help`
```

Comment:
```suggestion
Once downloaded, an informative help section is available using `nuctl --help`
```
This is the global help, not per command.

Maybe you mean `nuctl <command> --help` ? Another sentence?

Reply:
No need to specify all available commands on readme as they are easily discoverable via the global help